server/api.js

```diff
@@ -9,12 +10,28 @@ router.get("/", (req, res) => {
 
 router.get("/clients", (req, res) => {
 	db.query("SELECT * from clients;")
-		.then((result) => res.json(result.rows))
-		.catch((e) => {
-			res.status(400).send(e);
-		});
+    .then((result) => res.json(result.rows))
+	.catch((e) => {
+		console.error(e);
+		res.sendStatus(400);
+	});
 });
 
+router.post("/clients", (req, res) => {
```

Comment:
Do we need to change the implementation of this POST `api/clients` endpoint?

Reply:
Good question! Do you mean that right now we have this endpoint but nowhere for Matt to actually add a new client manually on the front end? 😕

Reply:
What I mean is that we seem to have deleted the existing implementation of POST `api/clients` (see [api.js#L99-L112](https://github.com/CodeYourFuture/scot-callback-cats/blob/main/server/api.js#L99-L112)) and replaced it with another implementation. If we don't need to change the implementation, can we revert to the old POST `api/clients` one that's on the `main` branch?